### DISCUSSION TITLE
How do I use it in Realms?

### DISCUSSION THREAD

**Discussion by roshikeymusica** [AvionBlock/VoiceCraft#106] [Q&A] [answered]
The wiki says it's available on Realms, but doesn't explain how to use it on Realms.

**Comment by Miniontoby** [comment, marked as answer]
Hi,

It is possible by using the Client Sided Positioning.

So you must then go into a creative world with cheats enabled before joining the Realm and then connect to the MCWSS server and then you can use Proximity Voicechat.

For more info on how to set that up, just watch the video on the MineDev channel. It is a little bit outdated as in it shows to install NET7, but you will need NET8 for the server.

I hope this was helpful
